modules/dag-l0/src/main/scala/org/tessellation/dag/l0/domain/snapshot/programs/SnapshotBinaryFeeCalculator.scala

```diff
@@ -0,0 +1,31 @@
+package org.tessellation.dag.l0.domain.snapshot.programs
+
+import org.tessellation.node.shared.config.types.FeeCalculatorConfig
+import org.tessellation.schema.GlobalSnapshotInfo
+import org.tessellation.schema.address.Address
+import org.tessellation.statechannel.StateChannelSnapshotBinary
+
+trait SnapshotBinaryFeeCalculator {
+  def calculateFee(binary: StateChannelSnapshotBinary, info: GlobalSnapshotInfo): Long
```

Comment:
shouldn't fee be `NonNegLong` or `PosLong`?

Reply:
changed